Windows Build

Hello,

Currently build scripts does not work on Windows.

Is there any plan for Windows support soon? or any hacks/tricks for now?

Thanks.

Tried few things.

This is the result of default build as "**npm run build**"

```
C:\Users\onur\Desktop\ckeditor5-build-classic-master>npm run build

> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 build C:\Users\onur\Desktop\ckeditor5-build-classic-master
> npm run create-entry-file && npm run build-ckeditor


> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 create-entry-file C:\Users\onur\Desktop\ckeditor5-build-classic-master
> ./bin/create-entry-file.js

'.' is not recognized as an internal or external command,
operable program or batch file.

npm ERR!     C:\Users\onur\AppData\Roaming\npm-cache\_logs\2017-10-12T10_13_33_333Z-debug.log
```
Later i changed create entry file script to this; "node bin/create-entry-file.js".

```
C:\Users\onur\Desktop\ckeditor5-build-classic-master>npm run build

> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 build C:\Users\onur\Desktop\ckeditor5-build-classic-master
> npm run create-entry-file && npm run build-ckeditor


> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 create-entry-file C:\Users\onur\Desktop\ckeditor5-build-classic-master
> node bin/create-entry-file.js

Creating the entry file...
Done.

> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 build-ckeditor C:\Users\onur\Desktop\ckeditor5-build-classic-master
> ./bin/build-ckeditor.sh

'.' is not recognized as an internal or external command,
operable program or batch file.
```

And eventually i checked shell script, saw webpack, and changed "build-ckeditor" command to "webpack".

This is the the error i got.

```
C:\Users\onur\Desktop\ckeditor5-build-classic-master>npm run build

> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 build C:\Users\onur\Desktop\ckeditor5-build-classic-master
> npm run create-entry-file && npm run build-ckeditor


> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 create-entry-file C:\Users\onur\Desktop\ckeditor5-build-classic-master
> node bin/create-entry-file.js

Creating the entry file...
Done.

> @ckeditor/ckeditor5-build-classic@1.0.0-alpha.1 build-ckeditor C:\Users\onur\Desktop\ckeditor5-build-classic-master
> webpack

C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\@ckeditor\ckeditor5-dev-webpack-plugin\lib\replacetcalls.js:29
                                const pathToCoreTranslationPackage = result.match( /.+\/ckeditor5-core/ )[ 0 ];
                                                                                                         ^

TypeError: Cannot read property '0' of null
    at compiler.resolvers.normal.resolve (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\@ckeditor\ckeditor5-dev-webpack-plugin\lib\replacetcalls.js:29:78)
    at onResolve (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:70:11)
    at innerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:143:22)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at next (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\tapable\lib\Tapable.js:249:35)
    at C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\UnsafeCachePlugin.js:39:22
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at innerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:143:22)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at next (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\tapable\lib\Tapable.js:249:35)
    at innerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:143:22)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at next (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\tapable\lib\Tapable.js:249:35)
    at resolver.doResolve.createInnerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\DescriptionFilePlugin.js:41:24)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at afterInnerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:165:22)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at next (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\tapable\lib\Tapable.js:249:35)
    at C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\ModuleKindPlugin.js:20:36
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at innerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:143:22)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at next (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\tapable\lib\Tapable.js:249:35)
    at innerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:143:22)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at next (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\tapable\lib\Tapable.js:249:35)
    at C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\forEachBail.js:29:14
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
    at innerCallback (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\Resolver.js:143:22)
    at callbackWrapper (C:\Users\onur\Desktop\ckeditor5-build-classic-master\node_modules\enhanced-resolve\lib\createInnerCallback.js:10:21)
```

I think we are now in same page with @pomek

--- a/package.json
+++ b/package.json
@@ -51,8 +51,8 @@
   },
   "scripts": {
     "build": "npm run create-entry-file && npm run build-ckeditor",
-    "create-entry-file": "./bin/create-entry-file.js",
-    "build-ckeditor": "./bin/build-ckeditor.sh",
+    "create-entry-file": "node bin/create-entry-file.js",
+    "build-ckeditor": "sh bin/build-ckeditor.sh",
     "preversion": "npm run build; if [ -n \"$(git status src/ckeditor.js build/ --porcelain)\" ]; then git add -u src/ckeditor.js build/ && git commit -m 'Internal: Build.'; fi"
   }
 }

